landlab/components/resource_redistribution/tests/test_resource_distribution.py

```diff
@@ -0,0 +1,78 @@
+"""
```

Comment:
Could you please add more extensive tests that better test the functioning of the component?

Reply:
done